gspread/client.py

```diff
@@ -207,31 +213,35 @@ def create(self, title, folder_id=None):
         spreadsheet_id = r.json()["id"]
         return self.open_by_key(spreadsheet_id)
 
-    def export_to_pdf(self, file_id):
-        """Export a spreadsheet to a PDF.
+    def export(self, file_id, format):
```

Comment:
please set the default value for `format` to `ExportFormat.PDF`, this is what was requested at first by the user and will be most likely the requested format.